From 1729cb28cb311bd326cb4495a51c86813270540e Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Tue, 2 Oct 2018 10:01:06 +0300
Subject: [PATCH 057/122] arm64: dts: ulcb-kf: increase SDIO frequency for WLAN
 chip

WL1837MOD supports up to 50 MHz on SDIO interface, increase max-frequency
value up to that.

This increases WLAN throughput.

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
---
 arch/arm64/boot/dts/renesas/ulcb-kf.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
index 61b7851..ccc84cb 100644
--- a/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb-kf.dtsi
@@ -1228,7 +1228,7 @@
 	no-1-8-v;
 	non-removable;
 	cap-power-off-card;
-	max-frequency = <26000000>;
+	max-frequency = <50000000>;
 	status = "okay";
 
 	#address-cells = <1>;
-- 
2.7.4

